corona-warn-app/cwa-app-android
# Contact journal extension duration picker (EXPOSUREAPP-5110)

Duration picker for contact journal.

How to test:
1) Go to test menu
2) Contact Diary Test Options
3) Click on "00:00" text field on top of the screen
4) Select any duration different from "00:00"
5) Check that text filed changed its value, changed color to black and now has blue border
6) Open dialog again
7) Select zero duration (00:00)
8) Check that text is now gray and don't have blue border

![Vuzix View_20210218_215906_670](https://user-images.githubusercontent.com/64849422/108407770-38fefc00-7235-11eb-9356-5a6d4ca7247f.jpg)
